minvws/nl-kat-coordination
# Update helper text for report names

### Changes
This PR adds a different helper text to the report name section (Generate Report > Step 4: Export setup).
The old text didn't include instructions about the placeholders that have been added.

The text only changes when "Recurrence - Yes, repeat" is selected.

### Demo
![afbeelding](https://github.com/user-attachments/assets/de34098a-0106-48cb-a1fa-2c93abe06968)


### QA notes
- Generate a report
- Select "Yes, repeat" in the Recurrence section
- Text should be visible

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
